tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -108,23 +114,11 @@ public async Task TestEndpointsReturnOk(TestEndpoints testEndpoints)
             }
 
             HttpResponseMessage? response = null;
-
-            using var client = app.CreateHttpClient(resource, null, clientBuilder =>
-            {
-                clientBuilder
-                    .ConfigureHttpClient(client => client.Timeout = Timeout.InfiniteTimeSpan)
-                    .AddStandardResilienceHandler(resilience =>
-                    {
-                        resilience.TotalRequestTimeout.Timeout = TimeSpan.FromMinutes(4);
-                        resilience.AttemptTimeout.Timeout = TimeSpan.FromSeconds(90);
-                        resilience.Retry.MaxRetryAttempts = 30;
-                        resilience.CircuitBreaker.SamplingDuration = resilience.AttemptTimeout.Timeout * 2;
-                    });
-            });
+            using var client = CreateHttpClientWithResilience(app, resource);
 
             foreach (var path in endpoints)
             {
-                _testOutput.WriteLine($"Calling endpoint '{client.BaseAddress}{path.TrimStart('/')} for resource '{resource}' in app '{Path.GetFileNameWithoutExtension(appHostPath)}'");
+                Console.WriteLine($"Calling endpoint '{client.BaseAddress}{path.TrimStart('/')} for resource '{resource}' in app '{Path.GetFileNameWithoutExtension(appHostPath)}'");
```

Comment:
Is this right? I would think it should stay _testOutput.

Reply:
Correct. I had changed it for debugging. It is useful when the whole thing times out, and so you don't get a .trx with the log. I'll drop it.